apis/serving/v1alpha1/predictor_types.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this was done by golang-lint, running `make fmt` (outside the dev container)

Reply:
@ckadner I'm curious why this would have just changed though... `make fmt` is run on all builds I thought?

Reply:
It only happens when I run `make fmt` locally on my Mac using Go 1.19

When I run `make fmt` inside the dev container (with Go 1.18), or `make run fmt` from my native Mac terminal, then the spaces won't get replaced. 

I found this comment on golangci/golangci-lint#3063 but not sure it's the same thing

> Due to a change of go fmt inside go1.19.
> We have to enforce the directive style.
>
> So the option allow-leading-space has been dropped.
>
> The syntax of directives is: [a-zA-Z]+:[a-zA-Z].*.
> The spaces around : and after the // must be removed.

Reply:
Ah, that is interesting. The [Go 1.19 Release Notes](https://tip.golang.org/doc/go1.19#go-doc) mention a change to `gofmt` that causes it to reformat godoc comments. Because the copyright headers are placed directly above the `package` declaration, `gofmt` for 1.19 is interpreting them as godocs and formatting them. (I verified that adding a blank line above the package directive prevents the change).

The effect that the change to `gofmt` has on our code will have to be addressed, but I'd prefer that it happens in a separate PR.

Reply:
agreed, I undid those changes for now

Reply:
I created issue #347 to track this